client/my-sites/email/email-management/home/email-mailbox-warnings.jsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button } from '@automattic/components';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'calypso/components/gridicon';
+import { resendVerificationEmail } from 'calypso/state/email-forwarding/actions';
+import {
+	getEmailForwardAddress,
+	hasGoogleAccountTOSWarning,
+	isEmailUserAdmin,
+	isGoogleEmailAccount,
+} from 'calypso/lib/emails';
+import { getGoogleAdminUrl } from 'calypso/lib/gsuite';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import {
+	EMAIL_WARNING_SLUG_GOOGLE_MAILBOX_TOS,
+	EMAIL_WARNING_SLUG_UNVERIFIED_FORWARD,
+} from 'calypso/lib/emails/email-provider-constants';
+import { isEmailForwardAccount } from 'calypso/lib/emails/is-email-forward-account';
```

Comment:
This is minor, but we could order those imports alphabetically so they are easier to parse.

Reply:
Done! I should have checked this earlier. 😳